lib/base-controller.js

```diff
@@ -23,27 +23,42 @@ function getErrorLength() {
   }
 }
 
-BaseController.prototype.getNextStep = function getNextStep(req, res) {
-  var next = Controller.prototype.getNextStep.apply(this, arguments);
+BaseController.prototype.saveValues = function saveValues(req) {
   var forks = (this.options || {}).forks || [];
 
   _.each(forks, function eachFork(fork) {
-    if (_.isFunction(fork.condition)) {
-      if (fork.condition(req, res)) {
-        next = req.baseUrl + fork.target;
-      }
-    }
     if (_.isPlainObject(fork.condition)) {
       if (fork.condition.value === req.form.values[fork.condition.field]) {
```

Comment:
It might be a good idea to check that `fork.condition` is an object or accessing ``value` or `.field` will throw an error